ebean-core/src/main/java/io/ebeaninternal/server/type/DataBind.java

```diff
@@ -46,12 +46,15 @@ public DataBind(DataTimeZone dataTimeZone, PreparedStatement pstmt, Connection c
 
   @Override
   public void pushJson(String json) {
+    assert this.json == null; // we can only push one value
```

Comment:
this may result in unexpected behaviour, if multiple beans are saved and a json from a previous bind is left on the "stack".